esphome/components/light/addressable_light_effect.h

```diff
@@ -143,14 +143,19 @@ class AddressableScanEffect : public AddressableLightEffect {
  public:
   explicit AddressableScanEffect(const std::string &name) : AddressableLightEffect(name) {}
   void set_move_interval(uint32_t move_interval) { this->move_interval_ = move_interval; }
+  void set_scan_width(uint32_t scan_width) { this->scan_width_ = scan_width; }
   void apply(AddressableLight &it, const ESPColor &current_color) override {
     it.all() = ESPColor::BLACK;
-    it[this->at_led_] = current_color;
+
+    for (auto i = 0; i < this->scan_width_; i++) {
+        it[this->at_led_ + i] = current_color;
```

Comment:
```suggestion
      it[this->at_led_ + i] = current_color;
```

https://travis-ci.org/esphome/esphome/jobs/541121316#L894